src/Stack.res

```diff
@@ -221,6 +217,10 @@ module Navigation = {
 
   @send external pop: (navigation, ~count: int=?, unit) => unit = "pop"
 
+  @send external popTo: (navigation, string) => unit = "popTo"
+  @send
+  external popToWithParams: (navigation, string, 'params) => unit = "popTo"
```

Comment:
With ReScript 11.1, we could have a single binding with an optional param (without trailing unit) instead.